mod/embed/views/default/elgg/embed.js

```diff
@@ -0,0 +1,189 @@
+/**
+ * Embed module bootstraps the embed funtionality
+ * Note that this module will initialize itself once elgg/ready module is loaded,
+ * hence you do not need to call elgg/embed#init after requiring elgg/embed module.
+ *
+ * @module elgg/embed
+ */
+define(function (require) {
+
+	var elgg = require('elgg');
+	var $ = require('jquery');
+	var lightbox = require('elgg/lightbox');
+	require('jquery.form');
+
+	elgg.provide('elgg.embed');
+
+	var embed = {
+		/**
+		 * Flag used to avoid duplicate event bindings
+		 */
+		ready: false,
+		/**
+		 * Initializes the module
+		 * @return void
+		 */
+		init: function () {
+			if (embed.ready) {
```

Comment:
just overwrites the function with `function (){}`, so calling it again does nothing.